zellij-tile/src/data.rs

```diff
@@ -24,6 +24,17 @@ pub enum Key {
     Esc,
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+
+pub enum Mouse {
+    ScrollUp(usize),   // number of lines
+    ScrollDown(usize), // number of lines
+    ClearScroll,
```

Comment:
I'd vote for removing `ClearScroll` probably (if you can't send it with a mouse)